### PR TITLE
Allowed spaces with wild cards in piratebay

### PR DESCRIPTION
**Pull request by PradumnUpadhyay**

### CODE REVIEW

**Comment by PradumnUpadhyay** [comment]
Use of proxychains or TBB is needed...

**Comment by TheSpeedX** [comment]
Better use Prawler or proxybroker

https://github.com/priyamharsh14/Prawler
or
https://github.com/constverum/ProxyBroker

File: torrgrab.py [hunk 1]
```diff
@@ -38,11 +37,11 @@ def scrapmagnet(site,se="pirate"):
 		return site
 def piratebay(term):
 	global name,link
-	term=term.replace(' ','+')
-	pblnk="https://indiaboat.art"
+	# term=term.replace(' ','+')
+	pblnk="https://pirateproxy.us/"
```
Comment (TheSpeedX) [inline torrgrab.py]: Use This instead @PradumnUpadhyay 

https://ahoythepirate.in/

Comment (PradumnUpadhyay) [reply torrgrab.py]: Okay but these links get ban after few weeks so we gotta continuously update em'

Comment (TheSpeedX) [reply torrgrab.py]: yup or else u add a bot to do that too xD

Comment (PradumnUpadhyay) [reply torrgrab.py]: creating one in nodejs xD...

Comment (PradumnUpadhyay) [reply torrgrab.py]: which will check if the site is up or not

**Pull request merged**